pkg/apis/migration/v1alpha1/miganalytic_types.go

```diff
@@ -0,0 +1,110 @@
+)
+
+// MigAnalyticSpec defines the desired state of MigAnalytic
+type MigAnalyticSpec struct {
+	Plan                *kapi.ObjectReference `json:"plan"`
+	AnalyzePVCapacity   bool                  `json:"analyzePVCapacity"`
+	AnalyzeImageCount   bool                  `json:"analyzeImageCount"`
+	AnalyzeK8SResources bool                  `json:"analyzeK8SResources"`
+	ListImages          bool                  `json:"listImages,omitempty"`
+	MaxListImages       int                   `json:"maxListImages,omitempty"`
+}
+
+// MigAnalyticStatus defines the observed state of MigAnalytic
+type MigAnalyticStatus struct {
+	Conditions
+	ObservedGeneration int64           `json:"observedGeneration,omitempty"`
+	Analytics          MigAnalyticPlan `json:"analytics,omitempty"`
+}
+
+// MigAnalyticPlan defines the observed state of MigAnalyticPlan
+type MigAnalyticPlan struct {
+	Plan                         string                 `json:"plan"`
+	PercentComplete              int                    `json:"percentComplete"`
+	K8SResourceTotal             int                    `json:"k8sResourceTotal"`
+	ExcludedK8SResourceTotal     int                    `json:"excludedk8sResourceTotal"`
+	IncompatibleK8SResourceTotal int                    `json:"incompatiblek8sResourceTotal"`
+	PVCapacity                   resource.Quantity      `json:"pvCapacity"`
+	PVCount                      int                    `json:"pvCount"`
+	ImageCount                   int                    `json:"imageCount"`
+	ImageSizeTotal               resource.Quantity      `json:"imageSizeTotal"`
+	Namespaces                   []MigAnalyticNamespace `json:"namespaces,omitempty"`
+}
+
+// MigAnalyticNamespace defines the observed state of MigAnalyticNamespace
+type MigAnalyticNamespace struct {
+	Namespace                    string                         `json:"namespace"`
+	K8SResourceTotal             int                            `json:"k8sResourceTotal"`
+	ExcludedK8SResourceTotal     int                            `json:"excludedK8SResourceTotal"`
+	IncompatibleK8SResourceTotal int                            `json:"incompatibleK8SResourceTotal"`
+	PVCapacity                   resource.Quantity              `json:"pvCapacity"`
+	PVCount                      int                            `json:"pvCount"`
+	ImageCount                   int                            `json:"imageCount"`
+	ImageSizeTotal               resource.Quantity              `json:"imageSizeTotal"`
+	Images                       []MigAnalyticNamespaceImage    `json:"images,omitempty"`
+	K8SResources                 []MigAnalyticNamespaceResource `json:"k8sResources,omitempty"`
+	ExcludedK8SResources         []MigAnalyticNamespaceResource `json:"excludedK8SResources,omitempty"`
+	IncompatibleK8SResources     []MigAnalyticNamespaceResource `json:"incompatibleK8SResources,omitempty"`
+}
+
+// MigAnalyticNamespaceResource defines the observed state of MigAnalyticNamespaceResource
```

Comment:
For some reason this was hidden in the github UI and I had to go searching for my ping, was this hidden because it resolved, or there's still an open question?

Can you give an example of the format you're asking about?

Reply:
I don't think it's resolved. It was hidden for me. I just saw you left a comment and was trying to find it...

Reply:
My 2 cents, +1 for consistency for sure. Would `groupKind` or `groupVersion` try to consolidate the two into a single field? Those are confusing to me without knowing the background around these suggestions. Also, what are the pros for the 2nd two over the first?

Reply:
I think the primary benefit for those suggestions was to condense the number of lines of output.

Reply:
> the number of lines of output

Is this really an optimization people care about?

Reply:
> Seems like there were some differing opinions on the format this should be presented in, I think this format is OK because it gives the user all of the info and the UI can mutate this info however it wants present it to the user.
> 
> @ibolton336 @eriknelson can you guys use the info in this format OK?

Is this replacing the incompatibleNamespaces list that is  currently pulled from the MigPlan.status.incompatibleNamespaces array?